src/System.Net.Http/tests/StressTests/HttpStress/Directory.Build.props

```diff
@@ -0,0 +1 @@
+<Project/>
```

Comment:
Are these files present to make sure the automated build system ignores building the sources?

Reply:
Exactly